configure.ac

```diff
@@ -226,13 +226,6 @@ AC_ARG_ENABLE([zmq],
   [disable ZMQ notifications])],
   [use_zmq=$enableval],
   [use_zmq=yes])
-AC_ARG_ENABLE([bip70],
```

Comment:
I think we probably want to leave this here, but turn ```--enable-bip70``` into an error. That way it's obvious to those who are currently manually overriding.

Reply:
Have reverted this removal, and now throw an error for `--enable-bip70`.